spec/order_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'order'
+
+describe "Order" do 
+
+  let (:menu){double :menu, dishes: {biriyani: 5.00, uk_curry: 6.00, cheeseburger: 4.00, pizza: 5.00, side_of_salmonella: 1.95}}
+  
+  before :each do
+    @subject = Order.new menu
```

Comment:
Use `described_class` instead of referencing the described class by name:

```ruby
# These do the same things
describe Order do
  Order.new
end
describe Order do
  described_class.new
end
```